src/styles/main.scss

```diff
@@ -1,3 +1,42 @@
-body {
+@import "./utils/variables";
+
+@import "./blocks/card.scss";
+@import "./blocks/header.scss";
+@import "./blocks/stars.scss";
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.body {
+  color: $colorText;
+  font-family: "Roboto", sans-serif;
```

Comment:
move it to the html selector

Reply:
Not fixed 
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/99403198/213871957-679c6350-80b5-42ce-8388-979313e6ba67.png">